plans/models.py

```diff
@@ -349,6 +349,21 @@ def create_for_users_without_plan(cls):
         return userplans
 
 
+@python_2_unicode_compatible
+class RecurringUserPlan(models.Model):
```

Comment:
Some docstring on this model would be great!

Explaining what it does and how it works and its role in the overall design